packages/core/content-manager/server/src/homepage/services/homepage.ts

```diff
@@ -147,72 +160,68 @@ const createHomepageService = ({ strapi }: { strapi: Core.Strapi }) => {
       model: uid,
     });
 
+  const queryLastDocuments = async (additionalQueryParams?: Record<string, unknown>) => {
```

Comment:
I think we could expose this on the content-manager homepage service. Then it would be accessible from review-workflows. You could do `strapi.plugin('content-manager').service('homepage').queryLastDocument()`